bundler/spec/install/binstubs_spec.rb

```diff
@@ -46,4 +46,44 @@ def Gem.bindir; "/usr/bin"; end
       )
     end
   end
+
+  describe "when env_shebang setting is not set" do
+    it "generates a system binstub with a /usr/bin/env shebang" do
+      install_gemfile <<-G
+        source "https://gem.repo1"
+        gem "myrack"
+      G
+
+      binstub_shebang = File.readlines(system_gem_path("bin/myrackup")).first
```

Comment:
Test failures are because Bundler 3 will no longer install to system path by default. You can instead use the `default_bundle_path` helper instead.